showcase/java/src/test/java/com/google/api/showcase/ShowcaseTest.java

```diff
@@ -258,4 +277,66 @@ public void block() {
                 .build());
     assertThat(result.getContent()).isEqualTo("Hello, World!");
   }
+
+  @Test
+  public void attemptSequence() {
```

Comment:
The other test is `TimeoutBackoff`, can you be more specific about what this test tests?

Reply:
I opted to remove this test because it wasn't doing anything more than the `TimeoutBackoff` one. It was the first one I added as I was developing it and I removed some others that were irrelevant to this PR, but neglected to remove this one.